tool/scripts/run_jetty_bash.twig

```diff
@@ -1,13 +1,14 @@
+#!/usr/bin/env bash
 "jre/bin/java" \
 -Xdebug "-agentlib:jdwp=transport=dt_socket,address=9999,server=y,suspend=n" \
--Xmx512M \
+-Xmx1024M \
 -XX:PermSize=64M \
 -XX:MaxPermSize=256M \
```

Comment:
Shall we remove this params?
https://stackoverflow.com/questions/18339707/permgen-elimination-in-jdk-8

Reply:
I've thought about it, but I'm not exactly sure how jazz handles this, because the values might be ignored by the jvm, but they show up in the [jazz UI](https://localhost:7443/jazz/admin#action=jazz.viewPage&id=com.ibm.team.repository.server). Haven't gotten around to testing what happens and seeing what the consequences are. I'm pretty sure I've used more than 512MB in the past without anything crashing, but I've decided to leave it in until I get around to dedicate a test to this.

Reply:
It does not hurt, so we can just leave it which makes it still compatible with Java 7...